openvpn/start.sh

```diff
@@ -4,9 +4,12 @@ if [ "$OPENVPN_PROVIDER" = "BTGUARD" ]
 then
 	echo "VPN PROVIDER: BTGUARD"
 	vpn_provider="btguard"
-else
+else if [ "$OPENVPN_PROVIDER" = "PIA" ]
```

Comment:
For backwards compatibility PIA should probably be the "default" VPN provider and any new ones should be called out explicitly.